bookwyrm/templates/book/edit/edit_book.html

```diff
@@ -52,19 +53,34 @@ <h2 class="title is-4">{% trans "Confirm Book Info" %}</h2>
                     {% for author in author_matches %}
                     <fieldset>
                         <legend class="title is-5 mb-1">
-                            {% blocktrans with name=author.name %}Is "{{ name }}" an existing author?{% endblocktrans %}
+                            {% blocktrans with name=author.name %}Is "{{ name }}" one of these authors?{% endblocktrans %}
                         </legend>
                         {% with forloop.counter0 as counter %}
+                        {% if author.isni_matches %}
+                        {% for isni_match in author.isni_matches %}
+                        <label class="label mt-2">
+                            <input type="radio" name="author_match-{{ counter }}" value="isni_match_{{ isni_match.isni }}" required>
+                            {{ isni_match.name }}
+                        </label>
+                        <p class="help ml-5 mb-2">
+                            <a href="{{ isni_match.uri }}" target="_blank">{{ isni_match.bio }}</a>
```

Comment:
For security, this link should have ` rel="noopener noreferrer"` since it opens an external link